examples/atari/dqfd/train_ddqn.py

```diff
@@ -0,0 +1,221 @@
+    parser.add_argument('--replay-start-size', type=int, default=5 * 10 ** 4,
+                        help='Minimum replay buffer size before ' +
+                        'performing gradient updates.')
+    parser.add_argument('--eval-n-steps', type=int, default=125000)
+    parser.add_argument('--eval-interval', type=int, default=250000)
+    parser.add_argument('--n-best-episodes', type=int, default=30)
+    parser.add_argument('--target-update-interval',
+                        type=int, default=10 ** 4,
+                        help='Frequency (in timesteps) at which ' +
+                        'the target network is updated.')
+    parser.add_argument('--update-interval', type=int, default=4,
+                        help='Frequency (in timesteps) of network updates.')
+    parser.add_argument('--minibatch-size', type=int, default=32)
+    parser.add_argument('--replay-buffer-size', type=int, default=10**6)
+    parser.add_argument('--num_step_return', type=int, default=10)
+    parser.set_defaults(clip_delta=True)
+    parser.add_argument('--no-clip-delta',
+                        dest='clip_delta', action='store_false')
+    args = parser.parse_args()
+
+    import logging
+    logging.basicConfig(level=args.logging_level)
+
+    # Set a random seed used in ChainerRL.
+    misc.set_random_seed(args.seed, gpus=(args.gpu,))
+
+    # Set different random seeds for train and test envs.
+    train_seed = args.seed
+    test_seed = 2 ** 31 - 1 - args.seed
+
+    args.outdir = experiments.prepare_output_dir(args, args.outdir)
+    print('Output files are saved in {}'.format(args.outdir))
+
+    def make_env(test):
+        # Use different random seeds for train and test envs
+        env_seed = test_seed if test else train_seed
+        env = atari_wrappers.wrap_deepmind(
+            atari_wrappers.make_atari(args.env, max_frames=None),
+            episode_life=not test,
+            clip_rewards=False)
+        env.seed(int(env_seed))
+        if test:
+            # Randomize actions like epsilon-greedy in evaluation as well
+            env = chainerrl.wrappers.RandomizeAction(env, 0.001)
+        else:
+            # Log scale train environment
+            env = LogScaleReward(env)
+        if args.monitor:
+            env = gym.wrappers.Monitor(
+                env, args.outdir,
+                mode='evaluation' if test else 'training')
+        if args.render:
+            env = chainerrl.wrappers.Render(env)
+        return env
+
+    env = make_env(test=False)
+    eval_env = make_env(test=True)
+
+    n_actions = env.action_space.n
+    q_func = DuelingDQN(n_actions)
+
+    # Draw the computational graph and save it in the output directory.
+    chainerrl.misc.draw_computational_graph(
+        [q_func(np.zeros((4, 84, 84), dtype=np.float32)[None])],
+        os.path.join(args.outdir, 'model'))
+
+    # Use the same hyperparameters as the Nature paper
+    opt = optimizers.RMSpropGraves(
```

Comment:
They do not. I assumed this since they take a lot of the parameters from the double-DQN paper, which states:
```The optimization employed to train the network is RMSProp (with momentum parameter 0.95).```
Does this indeed correspond to optimizers.RMSpropGraves with alpha=0.95?